lib/plugins/chat.js

```diff
@@ -173,7 +173,7 @@ function inject (bot, options) {
     return new Promise((resolve, reject) => {
       const resolveMessages = args.flatMap(x => x)
       function messageListener (msg) {
-        if (resolveMessages.some(x => msg === x)) {
+        if (resolveMessages.some(x => msg === x || (x instanceof RegExp ? x.test(msg) : false))) {
```

Comment:
how about if we combine the two into the latter ternary operator like this?
`        if (resolveMessages.some(x => x instanceof RegExp ? x.test(msg) : msg === x)) {`

Reply:
good idea!